apps/emqx_dashboard/src/emqx_dashboard_swagger.erl

```diff
@@ -0,0 +1,226 @@
+-module(emqx_dashboard_swagger).
+
+-include_lib("typerefl/include/types.hrl").
+-include_lib("hocon/include/hoconsc.hrl").
+-import(emqx_schema, [duration/0, duration_s/0, duration_ms/0, file/0, percent/0, ip_port/0]).
+
+%% API
+-export([spec/1, get_spec/2, get_spec_ref/2]).
+-export([translate_req/2]).
+-define(METHODS, [get, post, put, head, delete, patch, options, connect, trace]).
+-define(REF(_M_, _F_), iolist_to_binary([to_bin(_M_), ".", to_bin(_F_)])).
```

Comment:
is it possible to use `:` as delimiter ?

Reply:
yes!